selfdrive/car/subaru/interface.py

```diff
@@ -28,6 +28,17 @@ def get_params(candidate, fingerprint=gen_empty_fingerprint(), has_relay=False,
     ret.steerRateCost = 0.7
     ret.steerLimitTimer = 0.4
 
+    if candidate == CAR.ASCENT:
+      ret.safetyModel = car.CarParams.SafetyModel.subaru
```

Comment:
This isn't necessary as it's set as the default above

Reply:
removed